src/routes/acquirerRoutes.ts

```diff
@@ -127,6 +132,34 @@ router.get("/:resourceID/:step", async (req: Request, res: Response) => {
   } else {
     backLink = `/acquirer/${resourceID}/start`;
   }
+
+  let returnedNotes, returnedNotesTitle;
+  if (formStep === "check") {
```

Comment:
there might be an opportunity to move this into middleware and keep the route cleaner.
Thoughts?

Reply:
Yeah, I wasn't really happy with how messy this made the `:step` route.

Originally I'd imagined that `/check` would be a completely separate route, because it has to do stuff that's quite different to any of the other routes.

I'm happy to split it out now if you think it'd help?

Reply:
Yeah it feels more status than a form step